modules/ROOT/pages/introduction/intro-connector-configuration-overview.adoc

```diff
@@ -0,0 +1,148 @@
+
+* The first item is a placeholder for the Amazon SQS access key.
+* The second item is a placeholder for the Amazon SQS secret key.
+
+Using property placeholders improves the maintainability of your applications because they enable you to change configuration values in one place. If you don't use property placeholders and configuration parameter values change, you must search for changed parameters in the connector configuration screens or configuration XML.
+
+To make the values for the placeholders available, create a YAML file (`.yaml`) or properties (`.properties`) file to contain the values, place the file in `src/main/resources` in the Mule project, and then reference the file from the connector configuration.
+
+To reference a file that stores property values in Studio:
+
+. In the connector global element *Choose Global Type* screen, search for the element called *Configuration properties*.
+. Select *Configuration properties*.
+. Click *OK*.
+. Click *…​* to the right of the *File* field and navigate to your YAML or properties file.
+
+To reference a file that stores property values when configuring a connector using XML:
+
+. Include a `<configuration-properties>` element inside the `<mule>` element.
+. Set the `file` parameter with your properties file name. For example:
+
+[source,xml,linenums]
+----
+<mule>
+  <configuration-properties
+    file="myConfiguration.yaml"
+    doc:name="Configuration properties"
+    />`
+  ...
+</mule>
+----
+
+For more information about using property placeholders, see xref:mule-runtime::create-and-configure-your-properties-file.adoc[Configuring Property Placeholders].
+
+[[test-connectivity]]
+=== Test the Connectivity of the Connector
+
+Both Studio and Flow Designer enable you to test the connection between a connector and its data source by clicking a button.
+
+In Studio, after you configure the required connection values in the *Global Element Properties* screen, click *Test Connection*:
+
+.Click the *Test Connection* button in Studio to test the connection
+image::introduction/intro-config-overview-studio-test.png[Test the connection in Studio]
+
+In Flow Designer, after you configure the required connection values in the *Configure Connection* screen, click *Test*:
+
+.Click the *Test* button in Flow Designer to test the connection
+image::introduction/intro-config-overview-fd-test.png[Test the connection in Flow Designer]
+
+[[reconnection-strategy]]
+=== Configure a Reconnection Strategy for the Connector
+
+When a Mule app starts, a connectivity test runs on connectors that must connect to an external server. If the test fails, the default behavior is to log a warning message and continue with the deployment of the app, instead of blocking the deployment.
+
+You can modify this behavior by setting the number of reconnection attempts to try after an initial failure and forcing the deployment to fail when all connection attempts fail.
```

Comment:
```suggestion
You can modify this behavior by setting the number of reconnection attempts after an initial failure, which forces the deployment to fail when all connection attempts fail.
```

Reply:
That is not correct. These are two different actions - setting the number of reconnection attempts and forcing failure when all connection attempts fail.

Reply:
Again, your explanation makes sense, and you should always overrule edits that change your intended meaning, after making sure that you do not need to make that more clear by tweaking the wording.